app/javascript/tests/patient/assessment/steps/AssessmentCompleted.test.js

```diff
@@ -20,66 +20,66 @@ describe('AssessmentCompleted', () => {
     const language = 'eng';
     const wrapper = getWrapper(language, contact);
     expect(wrapper.find(Card.Header).exists()).toBeTruthy();
-    expect(wrapper.find(Card.Header).text()).toEqual(mockTranslations[language]['web']['title']);
+    expect(wrapper.find(Card.Header).text()).toEqual(mockTranslations[`${language}`]['web']['title']);
```

Comment:
same question as my first one

Reply:
generic object sink warning. These files hadn't been included in the linting before.